packages/fxa-auth-server/lib/senders/renderer/bindings.ts

```diff
@@ -2,6 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import { string } from '@hapi/joi';
```

Comment:
Unused import

Reply:
Oooo good catch, oops this was from me typing "string" for Typescript and VSCode automatically imported this and I didn't realize it.